src/strucclustutils/structcreatedb.cpp

```diff
@@ -142,79 +144,107 @@ void findInterfaceResidues(GemmiWrapper &readStructure, std::pair<size_t, size_t
                 y2 = readStructure.cb[res2Idx].y;
                 z2 = readStructure.cb[res2Idx].z;
             }
-            float D2 = 0;
-            D2 += (x1 - x2) * (x1 - x2);
-            D2 += (y1 - y2) * (y1 - y2);
-            D2 += (z1 - z2) * (z1 - z2);
-            float distance = sqrt(D2);
-            if (distance < shortestDistance) {
-                shortestDistance = distance;
+            // float D2 = 0;
+            // D2 += (x1 - x2) * (x1 - x2);
+            // D2 += (y1 - y2) * (y1 - y2);
+            // D2 += (z1 - z2) * (z1 - z2);
+            // float distance = sqrt(D2);
+            float distance = BasicFunction::dist(x1, y1, z1, x2, y2, z2);
```

Comment:
Please move to `MathUtil` and don't include a tmalign header

Reply:
yep!